src/test/java/uk/gov/hmcts/cmc/claimstore/processors/MixMapperTest.java

```diff
@@ -0,0 +1,103 @@
+package uk.gov.hmcts.cmc.claimstore.processors;
+
+import org.json.JSONException;
+import org.junit.Test;
+import uk.gov.hmcts.cmc.claimstore.config.JacksonConfiguration;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+import uk.gov.hmcts.cmc.domain.models.otherparty.OrganisationDetails;
+import uk.gov.hmcts.cmc.domain.models.otherparty.TheirDetails;
+import uk.gov.hmcts.cmc.domain.models.party.Organisation;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleClaimData;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleParty;
+import uk.gov.hmcts.cmc.domain.models.sampledata.SampleTheirDetails;
+
+import static uk.gov.hmcts.cmc.ccd.assertion.Assertions.assertThat;
+
+public class MixMapperTest {
```

Comment:
Please move the tests in ccd-adapter modules as well.

Reply:
I don't see that. This is all you need. 
```
    private JsonMapper processor = new JsonMapper(new CCDAdapterConfig().ccdObjectMapper());
```

Reply:
Probably time to move `JsonMapper` in common place. We should have the tests in same module where we have code.